test/group-2.test.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Tests group2
+ *
+ * @group group2
```

Comment:
Agreed, and since these tests are solely for demonstrative purposes regarding the test-matrix approach I'm proposing, they won't survive to see the functional PR. 👍